sources/RuleRunner.cpp

```diff
@@ -978,10 +978,10 @@ CommandAction(const BMessage& action, entry_ref& ref)
 
 	int result = system(value.String());
 	if (result) {
-		printf("\tTerminal Command: %s\n\t\tPossible error: "
+		printf("\Shell Command: %s\n\t\tPossible error: "
 			"command returned %d\n", value.String(), result);
 	} else
-		printf("\tTerminal Command: %s\n", value.String());
+		printf("\Shell Command: %s\n", value.String());
```

Comment:
Are you sure you wanted to escape the \S? Didn't you mean to leave the \t there?